### PR TITLE
Implement cleanup step after IPA build process

### PR DESCRIPTION
**Pull request by Rozzii**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -144,6 +144,9 @@ integration_test: ## Run integration test
 build_ipa:
 	$(CURDIR)/ci/scripts/image_scripts/start_centos_ipa_build.sh
 
+.PHONY: clean_ipa_builder
+clean_ipa_builder:
+	$(CURDIR)/ci/scripts/image_scripts/delete_ipa_builder.sh
```
Comment (furkatgofurov7) [inline Makefile:149]:
```suggestion
	$(CURDIR)/ci/scripts/image_scripts/clean_ipa_builder.sh
```

File: Makefile [hunk 2]
```diff
@@ -144,6 +144,9 @@ integration_test: ## Run integration test
 build_ipa:
 	$(CURDIR)/ci/scripts/image_scripts/start_centos_ipa_build.sh
 
+.PHONY: clean_ipa_builder
```
Comment (fmuyassarov) [inline Makefile:147]:
```suggestion
.PHONY: clean_ipa_builder_vm
```

File: ci/scripts/image_scripts/clean_ipa_builder.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env bash
+
+set -eu
+
+BUILDER_VM_NAME="${VM_NAME}"
```
Comment (fmuyassarov) [inline ci/scripts/image_scripts/clean_ipa_builder.sh:5]: not sure why we need this intermediate `BUILDER_VM_NAME` var? We could use directly `VM_NAME`?

**Review by fmuyassarov — CHANGES_REQUESTED** [review]

**Review by fmuyassarov — APPROVED** [review]

**Pull request merged**